web/views/errors/list.rss

```diff
@@ -61,7 +61,7 @@
 %    end
     </description>
     <category>{{res["item"]}}</category>
-%    url = 'http://%s/%s/map/#%s&zoom=13&lat=%s&lon=%s&level=%s&tags=&fixable=' % (website, lang, query, lat, lon, res["level"])
+%    url = 'http://%s/%s/map/#zoom=16&lat=%s&lon=%s&level=%s&tags=&fixable=&errorId=%s' % (website, lang, lat, lon, res["level"], res["uuid"])
```

Comment:
Need to remove `query` ?

Reply:
Good catch ! I removed it temporarily as it resulted in two `zoom` fragment parameters being present : one from query, one hardcoded. Restored it in 6efcd36.

That raises the following question : which zoom should be used in those kind of links ?

- The one from the user at the risk that the popup couldn't be opened ?
- The hardcoded high zoom to allow issues to be shown in dense areas ?

There is a third solution that seem a bit too complicated to me : pass the `marker_uuid` to the API through the Marker VectorGrid so that the API will always return it even if the query `LIMIT` is reached (with something like `ORDER BY marker.uuid = marker_uuid` DESC).

Reply:
I don't think that's blocking for this PR